src/main/java/by/Variables.java

```diff
@@ -0,0 +1,14 @@
+package by;
+
+import org.openqa.selenium.By;
+
+public class Variables {
+    public static By loginButton = By.linkText("Log in");
+    public static By loginField = By.name("login");
+    public static By passwordField = By.id("passp-field-passwd");
+    public static By emailsCheckbox = By.className("mail-Toolbar-Item-Checkbox");
+    public static By writeEmailButton = By.partialLinkText("Напи");
+    public static By forwardEmailIcon = By.xpath("//div[@data-key='view=toolbar-button-forward&threaded=yes&current_folder=1&sort_type=date&with_pins=yes&layout=messages&id=forward']");
```

Comment:
in order not to write such a long and complex locator, you can use `contains()`. Ex.:
`//div[contains(@data-key, 'forward')]`
define the difference between `//div[contains(@data-key, 'forward')]` and `.//div[contains(@data-key, 'forward')]`